app/organization/members/edit/controller.js

```diff
@@ -1,5 +1,21 @@
 import Ember from 'ember';
 
 export default Ember.Controller.extend({
-});
+  isSecurityOfficer: Ember.computed('organization.data.links.securityOfficer', 'model.data.links.self', function() {
```

Comment:
If we find ourselves doing this a bunch, we should make a util computed something like `computedBoundEqual`:

```javascript
// app/macros/bound-equal.js

let { get } = Ember;

export default function(dependentKey, comparisonKey) {
  return Ember.computed(dependentKey, comparisonKey, function(){
    return get(this, dependentKey) === get(this, comparisonKey);
  });
}
```